src/org/pentaho/reporting/platform/plugin/output/CachingPageableHTMLOutput.java

```diff
@@ -282,6 +295,12 @@ public IReportContent getCachedContent( final String key ) {
     return cache.get( key );
   }
 
+  public Map<String, Serializable> getCachedMetaData( final String key ) {
```

Comment:
THere is no reason to have this method as public, I suppose. No one can call this method from elsewhere.